LICENSE-MIT

```diff
@@ -0,0 +1,23 @@
+Permission is hereby granted, free of charge, to any
```

Comment:
The copyright notice is missing, is that intended?

Reply:
I refer to rustwide's license, it's also missing https://github.com/rust-lang/rustwide/blob/master/LICENSE-MIT
Should we have the notice?

Reply:
This is consistent with the other repos as well. e.g. [rust-lang](https://github.com/rust-lang/rust/blob/master/LICENSE-MIT)

Reply:
Oh, okay, I guess it's fine then. I'm wondering how you're supposed to comply with it though.